post-fs-data.sh

```diff
@@ -35,12 +42,34 @@ if [ -f /system/etc/gps.conf ]; then
   if grep -q "^SUPL_HOST=.*" "${MODDIR}/system/etc/gps.conf"; then
     # replace if it's already there
     sed -i "s/^SUPL_HOST=.*/SUPL_HOST=${SUPL_HOST}/" "${MODDIR}/system/etc/gps.conf"
+    sed -i "s/^SUPL_PORT=.*/SUPL_PORT=${SUPL_PORT}/" "${MODDIR}/system/etc/gps.conf"
   else
     # append if it's not there yet
     echo "SUPL_HOST=${SUPL_HOST}" >> "${MODDIR}/system/etc/gps.conf"
+    echo "SUPL_HOST=${SUPL_PORT}" >> "${MODDIR}/system/etc/gps.conf"
   fi
 
   sed -i "s/^NTP_SERVER=.*/NTP_SERVER=${NTP_SERVER}/" "${MODDIR}/system/etc/gps.conf"
+fi
+
+# For devices where only gps_debug is present
+if [ -f /system/etc/gps_debug.conf ] && [ ! -f /system/etc/gps.confg ]; then
```

Comment:
Is `gps.confg` a typo here?

Reply:
No, the script will only patch gps_debug.confg if gps.conf isn't found.

Reply:
Yeah, my bad